frontend/src/pages/servers/[ip].tsx

```diff
@@ -116,6 +128,20 @@ const Info = ({ data }: { data: All }) => (
                 <StatLabel>Language</StatLabel>
                 <StatNumber>{data.core.la}</StatNumber>
               </Stat>
+              <Link href={`samp://${data.dm ?? data.ip}`} isExternal>
```

Comment:
Minor: This could be extracted into its own component - `<ServerLink address="...">Quick Join</ServerLink>`